gnucash/import-export/aqb/dialog-ab-trans.c

```diff
@@ -388,6 +403,45 @@ gnc_ab_trans_dialog_new(GtkWidget *parent, GNC_AB_ACCOUNT_SPEC *ab_acc,
                            _("Originator BIC (Bank Code)"));
         break;
 
+#if (AQBANKING_VERSION_INT >= 60400)
+    case SEPA_INTERNAL_TRANSFER:
+        gtk_label_set_text(GTK_LABEL (heading_label),
+                           /* Translators: Strings from this file are
+                              needed only in countries that have one of
+                              aqbanking's Online Banking techniques
+                              available. This is 'OFX DirectConnect'
+                              (U.S. and others), 'HBCI' (Germany),
+                              or 'YellowNet' (Switzerland). If none of
+                              these techniques are available in your
+                              country, you may safely ignore strings
+                              from the import-export/hbci
+                              subdirectory. */
```

Comment:
We need this comment only once in the program. Since IIUC SEPA transfers are only available to FinTS users it might make sense to have a comment on all SEPA transfer strings saying something like `/* Translators: SEPA transfers available only in Germany. */`.

@fellen, the existing instance of this comment might be better worded "if no users of your language are likely to live in a country whose banks use these protocols you may safely leave these strings untranslated.". Also shouldn't we globally replace HBCI with FinTS in user-facing strings?

Reply:
Line numbers in this comment are from my last msgmerge.
> We need this comment only once in the program. 

gnucash/import-export/aqb/dialog-ab-trans.c:379 
gnucash/import-export/aqb/gnc-ab-utils.c:1141 have similar comments
> Since IIUC SEPA transfers are only available to FinTS users

EBICS is also available in the Single Euro Payments Area (SEPA) and also bidirectional (Upload of orders, download of statements). AFAIK by the banks in DE it is usually only offered to business owners. So we have not much knowledge about it here in the GnuCash project. Main languages of EBICS countries: de, fr, it.

@bayerstefan76 Is YellowNet still supported? I thougth it was dropped because of their `security by obscurity` policy.

IMHO we should insert sections about file formats and online banking methods into the glossary. There we have also a good place for comments.
> shouldn't we globally replace HBCI with FinTS in user-facing strings?

In most cases, but keep:
" * FinTS HBCI: You have to exchange the public parts of asymmetric keys with "
"execute the command line program \"aqhbci-tool\" for your account, as "
"follows: aqhbci-tool4 getaccsepa -b %s -a %s"

Replace by 'online banking':
"If active, the window will be closed automatically when you finish the HBCI/"
"(AqBanking or HBCI) Setup. After that, try again to download the Online "
Several occurrences of "HBCI/AqBanking" too. That was the switch from openHBCI to aqbanking. 

"Initial setup of Online Banking access (HBCI, or OFX DirectConnect, using " is missing EBICS,Paypal

"Verbose HBCI debug messages" verify for which it is really.

Several comments need review, too.
But this changes should be done in a separate PR.

Reply:
> But this changes should be done in a separate PR.

Of course. For this PR only fix the translator comments concerning this PR's strings.

Reply:
> EBICS is also available in the Single Euro Payments Area (SEPA) and also bidirectional (Upload of orders, download of statements). AFAIK by the banks in DE it is usually only offered to business owners. So we have not much knowledge about it here in the GnuCash project. Main languages of EBICS countries: de, fr, it.

OK, but isn't FinTS the only way that AQBanking supports it, and isn't that available only to customers of German banks?

Reply:
Sorry, I am only involved in the FinTS part of aqbanking, so for any other parts (inlcuding YelloqNet) you probably need to ask somebody else.

Reply:
Anyway, I removed the duplicated comment.

Reply:
@cstim Do you remember in which way EBICS is usable in Gnucash?